bin/ai/compat_14.nut

```diff
@@ -8,3 +8,29 @@
 /* This file contains code to downgrade the API from 15 to 14. */
 
 AIBridge.GetBridgeID <- AIBridge.GetBridgeType;
+
+function Text(text)
```

Comment:
I'm not sure how namespacing and scope works here, but is the simple name "Text" possible to conflict with anything a script author has used?